### ISSUE TITLE
Remove Default DHCP Options Set

### ISSUE THREAD

**Issue by sumodirjo** [gruntwork-io/cloud-nuke#118]
I'm trying this tool and can cleanly remove defaults VPCs on all regions but the default DHCP options are not removed. Are there any reason not to remove DHCP option set? Thank you

**Comment by brikis98** [comment]
@bwhaley I believe you added the default VPC clean up code. Could you answer this one?

**Comment by marinalimeira** [comment]
@bwhaley pinging you again :)

**Comment by bwhaley** [comment]
There's no reason not to do it. I've just confirmed that if you delete a default VPC and the default DHCP options, then recreate the default VPC (`aws ec2 create-default-vpc`), it will also recreate the DHCP options. Should be relatively straightforward to it.

**Comment by ina-stoyanova** [comment]
Thanks for looking into this Ben! I'll update the labels and queue it to be ready for picking up.

**Comment by hongil0316** [comment]
Something to note. While I was testing vpc nuking operation, I noticed: 
* After disassociating VPC from DHCP options, when I tried to delete it, the first attempt failed. 
* When I reran to delete the DHCP option, it went through. 

I'm thinking we should leave the current logic to disassociate DHCP option from VPC when nuking vpc and creating another resource types in cloud-nuke to clean up left over DHCP options?
